### DISCUSSION TITLE
The Chinese document no Chinese?

### DISCUSSION THREAD

**Discussion by gyt95** [parcel-bundler/parcel#7805] [General] [answered]
https://v2.parceljs.cn/docs/

This document is written in Chinese, but it is all in English, which seems to be no different from the English official website. Why?

**Comment by mischnic** [comment, marked as answer]
That domain is not affiliated with the Parcel project itself. So you have to ask the owner of that website

**Reply by gyt95** [reply]
> That domain is not affiliated with the Parcel project itself. So you have to ask the owner of that website

Thanks.
